assets/scss/components/stencil/productView/_productView.scss

```diff
@@ -372,6 +372,8 @@
     border: $productOptions-list-border;
     list-style: none;
     margin: 0;
+    max-height: 400px;
```

Comment:
How does this look on mobile or tablets. Can we confirm before we merge this one.

Reply:
<img width="247" alt="Screenshot 2020-07-06 at 11 33 01" src="https://user-images.githubusercontent.com/66319629/86573085-8a939880-bf7c-11ea-9821-73e500f655bf.png">
<img width="769" alt="Screenshot 2020-07-06 at 11 33 22" src="https://user-images.githubusercontent.com/66319629/86573090-8b2c2f00-bf7c-11ea-87e0-35970dfcd143.png">